tests/sentry/backup/test_models.py

```diff
@@ -217,6 +228,74 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Actor)
+    def test_actor(self):
+        self.create_user(email="test@example.com")
+        self.create_team(name="pre save team", organization=self.organization)
+        return self.import_export_then_validate()
+
+    @targets_models(ApiAuthorization)
+    def test_apiAuthorization(self):
```

Comment:
nit: think we should maintain snake_case for Model names, so this would be `test_api_authorization`. Same for the other test names below.